account_credit_control/models/credit_control_line.py

```diff
@@ -84,6 +84,14 @@ class CreditControlLine(models.Model):
         readonly=True,
         states={'draft': [('readonly', False)]},
     )
+    commercial_partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Commercial Entity',
+        compute_sudo=True,
+        related='partner_id.commercial_partner_id',
+        store=True,
+        readonly=True,
```

Comment:
No need of `readonly=True,` since it's the default behavior for a related field in 12.0.

```suggestion
```